index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Cooking with Lina</title>
+	<header><h1>Cooking with Lina<h1></header>
+</head>
+<body>
+	<p>Hello, and welcome to my website about cooking!</p>
+	<p>Cooking is one of the best hobbies! There are so many great things about cooking, so if you're looking for a new hobby, you should try cooking!</p>
+	
+	<section>
+	<h2>Reasons Cooking is Awesome:</h2>
+		<ul>
+			<li>You can cook healthy foods for yourself and your family.</li>
+			<li>You can spend time with your family and use it as a bonding activity.</li>
+			<li>You can slightly tweak and alter recipes so that they are flavored exactly the way you want!</li>
+		</ul>
+	</section>
+
+	<section>
+	<h2>Conversion Table</h2>
+		<p>Sometimes recipes will give you measurements, but your spoons and cups are in a different measurement. This table may help you with converting teaspoons to milliliters.</p>
+		<p>1 teaspoon is equal to 4.929 Milliliters, so you can calculate the conversion by multiplying the number of teaspoons by 4.929, as shown below.</p>
+		<table border="1">
+			<tr>
+				<th rowspan="4">Conversion Table</th>
+				<th>Tsp</th>
+				<th>Milliliter</th>
+			</tr>
+			<tr>
+				<td>1</td>
+				<td>4.9289</td>
+			</tr>
+			<tr>
+				<td>2</td>
+				<td>9.858</td>
+			</tr>
+			<tr>
+				<td>3</td>
+				<td>14.787</td>
+			</tr>
+		</table>
+	</section>
+
+	<section>
+	<h2>Pappardelle With Crispy Fennel and Fresh Herbs</h2>
+		<figure>
+			<img src="pappardelle.jpg" alt="pappardelle">
+			<figcaption>Recipe from Food52 - <a href="https://food52.com/blog/8233-pappardelle-with-crispy-fennel-fresh-herbs">Link to Recipe</a> </figcaption>
+		</figure>
+	</section>
+
+	<section>
+	<h2>Survey</h2>
+		<p>Thanks for visiting my website! Please take a moment to fill out this form...not that the information is being saved anywhere ;-)</p>
+		<form action="form_submit.html" method="post">
```

Comment:
is form_submit.html a placeholder for now?check out httpbin as an end point to test your form submission.